planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/src/scene.cpp

```diff
@@ -570,6 +570,24 @@ void StaticObstacleAvoidanceModule::fillEgoStatus(
     return;
   }
 
+  auto registered_sl_force_deactivated =
```

Comment:
```suggestion
  const auto registered_sl_force_deactivated =
```

Reply:
Fixed in 22fa38d5b09c711d4e91948907a4f7af5bcbc193